source/load-balancers/load-balancers.rst

```diff
@@ -0,0 +1,564 @@
+error that the driver is not configured to use it properly.
+
+If the :code:`loadBalanced=true` connection string option is not specified, the
+driver MUST omit the option in connection handshakes.
+
+*Example:*
+
+Driver connection string contains :code:`loadBalanced=false` or no
+:code:`loadBalanced` option:
+
+.. code:: typescript
+
+    { hello: 1 }
+
+Driver connection string contains :code:`loadBalanced=true`:
+
+.. code:: typescript
+
+    { hello: 1, loadBalanced: 1 }
+
+**Data-Bearing Server Type**
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A :code:`ServerType` of :code:`LoadBalancer` MUST be considered a data-bearing server.
+
+
+**Driver Sessions**
+-------------------
+
+**How To Check Whether A Deployment Supports Sessions**
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Having verified in step 1 that the :code:`TopologyDescription` includes at least one
+connected server a driver can now determine whether sessions are supported by inspecting
+the :code:`TopologyType` and :code:`logicalSessionTimeoutMinutes` property. When the
+:code:`TopologyType` is :code:`LoadBalanced`, sessions are always supported.
+
+
+**Server Selection**
+--------------------
+
+Topology type: Load Balanced
+
+A deployment of topology type Load Balanced contains one server of type :code:`LoadBalancer`.
+
+For read and write operations, the single server in the topology MUST always be selected.
+
+For read operations the :code:`mode`, :code:`tag_sets`, and :code:`maxStalenessSeconds`
+MUST be passed through to the load balancer but do not affect selection.
+See `Passing read preference to mongos <https://bit.ly/2PbyV0B>`__.
+
+
+**Connection Pooling**
+----------------------
+
+When the driver is in load balancing mode and executing any cursor-initiating command, the driver
+MUST NOT check the connection back into the connection pool until the initial cursor
+or :code:`getMore` command has returned with an :code:`id` of :code:`0`. Otherwise the
+driver MUST continue to use the same connection for all :code:`getMore` and
+:code:`killCursors` commands for a given cursor. Drivers MUST check the connection back into
+pool on all :code:`cursor.close()` calls.
+
+For drivers that implement a connection pool and when using a pinned connection, the driver MUST
+emit only 1 :code:`ConnectionCheckOutStartedEvent`, and only 1 :code:`ConnectionCheckedOutEvent`
+or :code:`ConnectionCheckOutFailedEvent`. Similarly, the driver MUST only publish 1
+:code:`ConnectionCheckedInEvent`.
+
+For single threaded drivers that do not use a connection pool, the driver MUST have only 1
+socket connection to the load balancer in load balancing mode.
+
+In the case of network errors for all operations that create a cursor, the driver MUST behave
+as defined in the Retryable Reads specification. If a getMore fails with a network error,
+drivers MUST unpin the connection that’s pinned to the cursor and return it to the pool
```

Comment:
We changed the rule about unpinning the connection after a network error on `getMore`. What do you think of this proposed wording:

> If a ``getMore`` fails with a network error, drivers MUST leave the connection pinned to the cursor. When the cursor is explicitly closed, drivers MUST NOT execute a ``killCursors`` command because the pinned connection is no longer valid and MUST return the connection back to the pool.

Reply:
I like that suggestion. Updated to that.